FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift

```diff
@@ -60,11 +60,6 @@ class ModelDownloadTask {
 }
 
 extension ModelDownloadTask {
-  /// Name for model file stored on device.
-  var downloadedModelFileName: String {
```

Comment:
Was this not used anywhere?

Reply:
This moved to ModelFileManager but I forgot to delete this (`getDownloadedModelFileURL`). It's not being used anywhere.